src/tests/mocks/handlers.ts

```diff
@@ -0,0 +1,92 @@
+import { rest } from 'msw';
+
+const API_BASE_URL = 'http://localhost:3001';
+
+//Mock data
+export let mockPosts = [
+  {
+    _id: '1',
+    task: 'Task 1',
+    completed: false,
+    createdAt: '2023-09-02T20:44:24.326Z',
+    updatedAt: '2023-09-02T20:44:24.326Z',
+  },
+  {
+    _id: '2',
+    task: 'Task 2',
+    completed: true,
+    createdAt: '2023-09-02T20:44:24.326Z',
+    updatedAt: '2023-09-02T20:44:24.326Z',
+  },
+  {
+    _id: '3',
+    task: 'Task 3',
+    completed: true,
+    createdAt: '2023-09-02T20:44:24.326Z',
+    updatedAt: '2023-09-02T20:44:24.326Z',
+  },
+];
+console.log('handlers ran');
```

Comment:
✨ Suggestion: get rid of this log; do you really need it?

Reply:
removed